README.md

```diff
@@ -14,6 +14,275 @@ To run this microservice, you must have SBT installed. You should then be able t
 ### Testing
 ```sbt test it:test```
 
+
+## Endpoints
+
+### GET /returns/vrn/{vrn}?period-key={periodkey}
+
+Where:
+* **vrn** is a valid VRN, for example: "444444444"
+* **periodkey** is the period key of the return, for example: "18AA"
+
+#### Success Response
+
+**HTTP Status**: 200
+
+**Example HTTP Response Body**:
+```
+{
+   "periodKey": "18AA",
+   "vatDueSales": 100.00,
+   "vatDueAcquisitions": 300.00,
+   "vatDueTotal": 400.00,
+   "vatReclaimedCurrPeriod": 150.00,
+   "vatDueNet": 250.00,
+   "totalValueSalesExVAT": 100.00,
+   "totalValuePurchasesExVAT": 100.00,
+   "totalValueGoodsSuppliedExVAT": 100.00,
+   "totalAllAcquisitionsExVAT": 100.00,
+   "agentReferenceNumber": "XAIT0000000000"
+}
+```
+Where:
+* **agentReferenceNumber** is an optional field
+
+#### Error Responses
+
+Status | Error codes
+:---:|:---:
+400    |INVALID_VRN, INVALID_PERIODKEY
+403    |NOT_FOUND_VRN, INVALID_IDENTIFIER, INVALID_INPUTDATA
+404    |NOT_FOUND
+500    |SERVER_ERROR
+503    |SERVICE_UNAVAILABLE
+
+
+### POST /returns/vrn/:vrn
```

Comment:
There is an additional header that needs to be sent for this POST. It's the `OriginatorID` that is a String